packages/editor/src/components/post-text-editor/style.scss

```diff
@@ -35,15 +40,12 @@ textarea.editor-post-text-editor {
 		position: relative;
 	}
 
-	&::-webkit-input-placeholder {
-		color: $dark-gray-placeholder;
-	}
-
-	&::-moz-placeholder {
-		color: $dark-gray-placeholder;
-	}
-
-	&:-ms-input-placeholder {
+	// Show placeholder when empty or only contains br tags
+	&:empty::before,
+	&:has(br:only-child)::before {
+		content: attr(data-placeholder);
 		color: $dark-gray-placeholder;
+		pointer-events: none;
+		position: absolute;
 	}
```

Comment:
The `pre` element can't have a `placeholder` attribute, so visually display a placeholder text as a pseudo-element.